roles/build_push_container/molecule/default/molecule.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Mainly used to override the defaults set in .config/molecule/
+# By default, it uses the "config_podman.yml" - in CI, it will use
+# "config_local.yml".
+log: true
+
+provisioner:
+  name: ansible
+  log: true
+  env:
+    ANSIBLE_STDOUT_CALLBACK: yaml
+
+prerun: false
+
+scenario:
+  test_sequence:
+    - converge
```

Comment:
hmm ok, so you don't want any other step. Might be OK, since the prepare would just have one task. Just "unusual", but I'm OK.